src/libs/NumberUtils.js

```diff
@@ -0,0 +1,27 @@
+import CONST from '../CONST';
+
+/**
+ * Generates a random positive 64 bit numeric string by randomly generating the left half and right half and concatenating them. Used to generate client-side ids.
+ * @returns {string} randomly generated 64 bit string
+ */
+function rand64() {
```

Comment:
We should suppress the lint error for now `/* eslint-disable no-unused-vars */`

Reply:
I was thinking about this but whoever uses the function later would have to remember to take out the eslint that we would add. I can add it though if you think it's better to do it this way.

Reply:
Yea, I think we should add it, otherwise tests would break for others.

Reply:
Created the issue here. I'll update the link once we merge this PR.

Reply:
@luacmartins there is no link on your "here" 😄

Reply:
Classic. Edited and added the link. Thanks!